### PR TITLE
util: define more emoji glyphs

### PR DESCRIPTION
**Pull request by poettering**
This isnt too useful on its own, but is preparation for #20156

### CODE REVIEW

File: src/basic/locale-util.c [hunk 1]
```diff
@@ -375,6 +375,10 @@ const char *special_glyph(SpecialGlyph code) {
                         [SPECIAL_GLYPH_DEPRESSED_SMILEY]        = ":-[",
                         [SPECIAL_GLYPH_LOCK_AND_KEY]            = "o-,",
                         [SPECIAL_GLYPH_TOUCH]                   = "O=",    /* Yeah, not very convincing, can you do it better? */
+
+                        [SPECIAL_GLYPH_RECYCLING]               = "~",
+                        [SPECIAL_GLYPH_DOWNLOAD]                = "\\",
```
Comment (yuwata) [inline src/basic/locale-util.c:379]: I am not sure that these two are good for fallback, but ok, as they are not used right now.

BTW, why do you add a blank line??

File: src/basic/locale-util.c [hunk 2]
```diff
@@ -421,7 +425,11 @@ const char *special_glyph(SpecialGlyph code) {
                         [SPECIAL_GLYPH_LOCK_AND_KEY]            = "\360\237\224\220",         /* 🔐 (actually called: CLOSED LOCK WITH KEY) */
 
                         /* This emoji is a single character cell glyph in Unicode, and two in ASCII */
-                        [SPECIAL_GLYPH_TOUCH]                   = "\360\237\221\206",         /* 👆 (actually called: BACKHAND INDEX POINTING UP */
+                        [SPECIAL_GLYPH_TOUCH]                   = "\360\237\221\206",         /* 👆 (actually called: BACKHAND INDEX POINTING UP) */
+
```
Comment (yuwata) [inline src/basic/locale-util.c:428]: Same here, I think it is not necessary to add a blank line. Or, these are belonging to different categories?

**Review by yuwata — APPROVED** [review]
LGTM.

**Comment by yuwata** [comment]
I set `-with-minor-suggestions` tag, but feel free to merge as is. That's super minor thing.

**Comment by keszybz** [comment]
LGTM too.

**Comment by poettering** [comment]
I dropped the empty lines, they had no purpose

**Pull request merged**